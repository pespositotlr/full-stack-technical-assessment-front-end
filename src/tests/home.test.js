import React from 'react';
import renderer from 'react-test-renderer';

describe('<Home /> component', () => {
    it("renders welcome text", () => {
        const wrapper = shallow(<Home />);
        const welcome = "Welcome to my Test Application!";
        expect(wrapper.contains(welcome)).toEqual(true);
    });

    it("renders welcome div", () => {
        const wrapper = shallow(<Home />);
        const welcome = <div className="homeText">
            Welcome to my Test Application!
        </div>;
        expect(wrapper.contains(welcome)).toEqual(true);
    });
    
    it("renders correctly", () => {
        const tree = renderer
            .create(<Home />)
            .toJSON();
        expect(tree).toMatchSnapshot();
    });
});


