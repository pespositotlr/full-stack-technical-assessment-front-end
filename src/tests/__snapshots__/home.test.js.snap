// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Home /> component renders correctly 1`] = `
<div
  className="row"
>
  <div
    className="col-md-12"
  >
    <div
      className="homeText"
    >
      Welcome to my Test Application!
    </div>
  </div>
</div>
`;
